dinersclub/main.go

```diff
@@ -96,6 +98,10 @@ func (dc *DC) Job(pe *PaymentEvent) error {
 
 	res := new(Result)
 	op := func() error {
+		e := provider.Auth(dc.pool, pe.Userid)
```

Comment:
Ah - sorry this wasn't clear, the userid in an event is not the userid of the reloadly account. We have this terrible naming convention (sorry!) where both the respondent and the survey creator are both called "users" in different places. 

In an event, the userid is the respondent. That doesn't help us. We need the pageid, which is the entity the respondent is chatting with, and from that we can lookup the survey creator's userid (by looking up the facebook page id in the credentals table.